nf_core/licences.py

```diff
@@ -61,7 +61,9 @@ def get_environment_file(self):
             response = requests.get(env_url)
             # Check that the pipeline exists
             if response.status_code == 404:
-                raise LookupError("Couldn't find pipeline conda file: {}".format(env_url))
+                raise LookupError(
+                    f"Couldn't find pipeline conda file: {env_url}. (Note: DSL2 pipelines are currently not supported by this command."
```

Comment:
Since we might have a locally installed DSL2 pipeline we should also warn about the missing support for DSL2 pipelines when the `environment.yml` file is not found in the local directory, right?

Reply:
Good point, thanks!